src/v/redpanda/admin_server.h

```diff
@@ -70,6 +71,50 @@ class admin_server {
               path, std::move(req), std::move(rep));
         }
     };
+
+    enum auth_level {
```

Comment:
enum class for strong type?

Reply:
As long as we're using it as a template arg, even the regular enum is strong enough to catch someone passing in a random integer.  However, `enum class` is the nice modern thing, so I've converted it over.